apps/web/styles/globals.css

```diff
@@ -1,3 +1,14 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+@layer components {
+  body {
```

Comment:
I don't think it's needed.

Thanks for working on it btw. I'll check the whole PR asap when I'm free.